tests_end_to_end/e2e_tests_utils.py

```diff
@@ -45,3 +48,65 @@ def connect_to_server(insecure: bool = False):
             "Could not connect to XSOAR server. Please check your connection configurations."
         )
     return client
+
+def create_pack(repo: Repo):
+    unique_id = random.randint(1, 100)
+    pack_name = "e2e_test_" + str(unique_id)
+    pack = repo.create_pack(name=pack_name)
+    source_pack_path = Path(pack.path)
+
+    return pack, pack_name, source_pack_path
+
+def create_playbook(pack, pack_name):
+    playbook_name = "pb_e2e_test_" + pack_name
+    playbook: Playbook = pack.create_playbook(name=playbook_name)
+    playbook.create_default_playbook(name=playbook_name)
+    playbook_path = Path(playbook.yml.path)
+    
+    return playbook, playbook_name, playbook_path
+
+def create_xif_file_modeling_rules(pack_path, modeling_rules_name, modeling_rules_string):
+
+    with open(f"{pack_path}/ModelingRules/{modeling_rules_name}/{modeling_rules_name}.xif", 'w') as file:
+        file.write(modeling_rules_string)
+
+def create_yml_file_modeling_rules(pack_path, modeling_rules_name, modeling_rules_id):
+    
+    yml_file_string = f"""
+fromversion: 8.4.0
+id: {modeling_rules_id}
+name: {modeling_rules_name}
+rules: ''
+schema: ''
+    """
+    
+    with open(f"{pack_path}/ModelingRules/{modeling_rules_name}/{modeling_rules_name}.yml", 'w') as file:
+        file.write(yml_file_string)
+
+def create_schema_file_modeling_rules(pack_path, modeling_rules_name, modeling_rules_schema_string):
+
+    with open(f"{pack_path}/ModelingRules/{modeling_rules_name}/{modeling_rules_name}_schema.json", 'w') as file:
+        file.write(modeling_rules_schema_string)
+
+def create_testdata_file_modeling_rules(pack_path, modeling_rules_name, test_data_string):
+
+    with open(f"{pack_path}/ModelingRules/{modeling_rules_name}/{modeling_rules_name}_testdata.json", 'w') as file:
+        file.write(test_data_string)
+
+def create_modeling_rules_folder(pack_path, modeling_rules_name, modeling_rules_id, modeling_rules_string, test_data_string, modeling_rules_schema_string):
+    """Creating the modeling rules folder, consists of modeling rules file, schema, yml and testdata file
+
+    Args:
+        pack_path (str): The path of the pack to add modeling rules to
+        modeling_rules_name (str): The name of the modeling rules.
+        modeling_rules_id (str): The ID of the modeling rules.
+        modeling_rules_string (str): The data of the modeling rules xif file.
+        test_data_string (str): The data of the testdata.
+        modeling_rules_schema_string (str): The data of the schema
+    """
+    cli(f"mkdir -p {pack_path}/ModelingRules/{modeling_rules_name}")
```

Comment:
```suggestion
    directory_path = Path("{pack_path}/ModelingRules/{modeling_rules_name}")
    directory_path.mkdir(parents=True, exist_ok=True)
```

do it more pythonic :), no?